docs/getting-started.rst

```diff
@@ -174,117 +158,71 @@ Note that if one of these columns exists, it will be overwritten.
 +-------------------+-----------------------------------------------------+------------+
-|abgst              |                                                     | Float      |
+|pensions_sim       |Monthly amount of old-age pensions                   | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|soli               |                                                     | Float      |
+|gross_e1           |Taxable Inc. from Self-Employment, individual        | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|soli_tu            |                                                     | Float      |
+|gross_e5           |Taxable Inc. from Capital, individual                | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|kindergeld_basis   |                                                     | Float      |
+|gross_e6           |Taxable Inc. from Rents, individual                  | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|kindergeld_tu_basis|                                                     | Float      |
+|gross_e7           |Taxable Inc. from Pensions, individual               | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|incometax_tu       |                                                     | Float      |
+|gross_e1_tu        |Taxable Inc. from Self-Employment, couple sum        | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|incometax          |                                                     | Float      |
+|gross_e4_tu        |Taxable Inc. from Earnings, couple sum               | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|kindergeld         |                                                     | Float      |
+|gross_e5_tu        |Taxable Inc. from Capital, couple sum                | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|kindergeld_hh      |                                                     | Float      |
+|gross_e6_tu        |Taxable Inc. from Rents, couple sum                  | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|kindergeld_tu      |                                                     | Float      |
+|gross_e7_tu        |Taxable Inc. from Pensions, couple sum               | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|uhv                |                                                     | Float      |
+|abgst_tu           |Monthly Capital Income Tax Due, couple sum           | Float      |
```

Comment:
Please be consistent in capitalisation.

I would actually suggest making everything lowercase, except for the very first letter.